site/components/TransactionsModal.js

```diff
@@ -0,0 +1,144 @@
+import useTranslation from 'next-translate/useTranslation'
+
+import { EtherscanLink } from './EtherscanLink'
+import JustifiedBetweenRow from './JustifiedBetweenRow'
+import Modal from './Modal'
+import SvgContainer from './svg/SvgContainer'
+import { useNumberFormat } from '../hooks/useNumberFormat'
+
+const TransactionModalRow = function ({ text, value, tipLink = '' }) {
+  return (
+    <JustifiedBetweenRow
+      keyComponent={
+        tipLink ? (
+          <a
+            className={'text-gray-350 text-sm flex justify-bewtween'}
```

Comment:
```suggestion
            className={'text-gray-350 text-sm flex justify-between'}
```

Reply:
Just found this because it's wrong where you copied from too 😅

Reply:
Good catch!

Reply:
Fixed in bdd2772.